theano/sandbox/cuda/tests/test_dnn.py

```diff
@@ -418,26 +416,20 @@ def test_pooling3d():
     if not cuda.dnn.dnn_available() or cuda.dnn.version() < (3000, 3000):
         raise SkipTest(cuda.dnn.dnn_available.msg)
 
-    mode_without_gpu_ref = theano.compile.mode.get_mode(
```

Comment:
Using this mode was explicit. We don't want the default mode that could be DebugMode for the reference. That would make the tests too slow in DebugMode. So add this back. Was there a reason why you removed it instead of just code clean up? Maybe add a comment so that the next person understand why it is there. Something like:

# We force the FAST_RUN as we don't want the reference to run in DebugMode.